pkg/codegen/internal/test/testdata/output-funcs-edgeorder/dotnet/ListConfigurations.cs

```diff
@@ -31,22 +32,22 @@ public static Output<ListConfigurationsResult> Invoke(ListConfigurationsInvokeAr
     public sealed class ListConfigurationsArgs : Pulumi.InvokeArgs
     {
         [Input("configurationFilters", required: true)]
-        private List<Inputs.ConfigurationFilters>? _configurationFilters;
+        private List<Pulumi.Myedgeorder.Inputs.ConfigurationFilters>? _configurationFilters;
```

Comment:
Ah, this is a case where you could just have `Inputs.ConfigurationFilters` here because this is already within `namespace Pulumi.Myedgeorder { ... }`.